src/components/Tabs.stories.tsx

```diff
@@ -161,6 +161,23 @@ export const TabsWithActions = () => {
   return <TabsWithContent tabs={testTabs} onChange={setSelectedTab} selected={selectedTab} ariaLabel="Sample Tabs" />;
 };
 
+export const OneTabWithActions = () => {
+  const [selectedTab, setSelectedTab] = useState("tab1");
+  const testTabs: Tab<TabValue>[] = [
+    {
+      name: "Tab 1",
+      value: "tab1",
```

Comment:
Should this tab be disabled to trigger the `hideTabs` line?

Reply:
No, as long as there is only one enabled tab, then the tabs will be hidden.

Reply:
Ah right! Thanks